### ISSUE TITLE
Just when I use forward got the message "$(...).getFormPrefix is not a function"

### ISSUE THREAD

**Issue by Fabitosoft** [yourlabs/django-autocomplete-light#806]
Hello,

I've been using this amazing library 3 month ago but today, when I tried to use forward in one o my fields there is a message that said "$(...).getFormPrefix is not a function". These are the two fields that I tried to use:
```

cliente_biable = forms.ModelChoiceField(
        queryset=Cliente.objects.all(),
        widget=autocomplete.ModelSelect2(url='biable:cliente-autocomplete'),
        required=False,
        label='Cliente CGuno'
    )

contacto = forms.ModelChoiceField(
        queryset=ContactoEmpresa.objects.all(),
        widget=autocomplete.ModelSelect2(url='contactos:contactos-autocomplete',forward=['cliente_biable ']),
        required=False,
        label='Contacto'
    )
```

At the moment that I took out forward in the last one, it works properly.

In the HTML field I put 

```
{% block contenct %}
     <form action="" method="post">
          {{ form.as_p }}
     </form>
{% endblock %}

{% block footer %}
    <script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
    {{ form.media }}
{% endblock %}
```

Have someone had the same problem? Is there any solution?
Thanks.

**Comment by jpic** [comment]
I don't understand, it should be defined by this script: https://github.com/yourlabs/django-autocomplete-light/blob/master/src/dal/static/autocomplete_light/autocomplete.init.js

Perhaps you need to run collectstatic again ? Do you have DEBUG enabled ?

Perhaps this document can help in addition to Django's documentation on static files: http://blog.yourlabs.org/post/30382323418/surviving-djangocontribstaticfiles-or-how-to

**Comment by Fabitosoft** [comment]
Hi,
You got a point there, I have noticed that JQuery has been loaded twice. 
First, when I made the call with media after my form, and after, because it is in the base.html. I have termporally removed it from base.html and it works properly. 
Does someone has any peace of code to resolve this? I mean, to keep the JQuery that I have in base.html but just in this case to change it for the other version? Any good solution to make more clear my script?

Thanks
